src/shared/components/app/navbar.tsx

```diff
@@ -10,17 +10,17 @@ import { i18n } from "../../i18next";
 import { UserService } from "../../services";
 import { HttpService, RequestState } from "../../services/HttpService";
 import {
-  amAdmin,
-  canCreateCommunity,
   donateLemmyUrl,
-  isBrowser,
   myAuth,
   numToSI,
-  poll,
   showAvatars,
   toast,
   updateUnreadCountsInterval,
 } from "../../utils";
+import { isBrowser } from "../../utils/browser/is-browser";
```

Comment:
You could export all these in an `index.ts` to avoid a ton of extra import lines, but I leave that up to you.

Reply:
I personally like the extra import lines, since it avoids having to maintain an additional `index.ts`. 🤷